notebooks/custom_kinfraglib/1_3_custom_filters_synthesizability.ipynb

```diff
@@ -0,0 +1,2349 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    fragment_library_synthesizability = filters.analysis.number_of_accepted(
</pre><p>This looks like a duplication to the lines above?</p><p> </p><p>-&gt; wait, I see below, it checks the number of accepted when combining filters, add a description above to make this clear.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/sonjaleo/KinFragLib/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>